packages/types/src/http/fulfillment/admin/fulfillment.ts

```diff
@@ -25,3 +25,12 @@ export interface AdminFulfillmentResponse {
   updated_at: Date
   deleted_at: Date | null
 }
+
+/**
+ * @experimental
+ */
+export interface AdminFulfillmentSetsDeleteResponse {
```

Comment:
question: Don't we have a generic delete response? I recall @adrien2p adding that.